src/main/java/org/shanerx/tradeshop/objects/Shop.java

```diff
@@ -36,10 +36,7 @@
 import org.bukkit.inventory.Inventory;
 import org.bukkit.inventory.ItemStack;
 import org.shanerx.tradeshop.TradeShop;
-import org.shanerx.tradeshop.enumys.Setting;
-import org.shanerx.tradeshop.enumys.ShopRole;
-import org.shanerx.tradeshop.enumys.ShopStatus;
-import org.shanerx.tradeshop.enumys.ShopType;
+import org.shanerx.tradeshop.enumys.*;
```

Comment:
Not a fan of this (wildcard imports)

Reply:
Neither am I, my settings in idea may be wonky. But also it's only for our own files so I wasn't too concerned about it.

Reply:
fixed in my idea and re-optimized all the imports so they should all be gone, messaged in discord for the settings that needed to be changed.